From: siretart <subversion@mplayerhq.hu>
Subject: r31058 - trunk/libvo/vo_directfb2.c
Newsgroups: gmane.comp.video.mplayer.cvs
To: mplayer-cvslog@mplayerhq.hu
Date: Fri, 23 Apr 2010 12:04:56 +0200 (CEST)
Reply-To: MPlayer CVS changelogs and discussion <mplayer-cvslog@mplayerhq.hu>

Author: siretart
Date: Fri Apr 23 12:04:56 2010
New Revision: 31058

Log:
include sys/kd.h on non-Linux architectures correctly

Modified:
   trunk/libvo/vo_directfb2.c

Modified: trunk/libvo/vo_directfb2.c
==============================================================================
--- trunk/libvo/vo_directfb2.c	Thu Apr 22 16:02:20 2010	(r31057)
+++ trunk/libvo/vo_directfb2.c	Fri Apr 23 12:04:56 2010	(r31058)
@@ -35,9 +35,9 @@
 #include <string.h>
 
 #ifdef __linux__
-#include <sys/kd.h>
-#else
 #include <linux/kd.h>
+#else
+#include <sys/kd.h>
 #endif
 
 #include "config.h"
